collab-database/src/database.rs

```diff
@@ -1157,30 +1197,22 @@ impl Database {
     })
   }
 
-  pub fn duplicate_database(&self) -> DatabaseData {
-    let inline_view_id = self.get_inline_view_id();
+  pub fn get_all_database_data(&self) -> DatabaseData {
```

Comment:
to_create_database_params is kind of vague. Only need to regenerate the ids when duplicating a database. Also DatabaseData should not contains biz logic. So i prefer using `duplicate_database`